templates/common/test_utils.go

```diff
@@ -47,26 +46,8 @@ var (
 	MinimalGitHeadShortSHA = MinimalGitHeadSHA[:7]
 )
 
-// CmpFileMode is a cmp option that handles the conflict between Unix and
-// Windows systems file permissions.
+// CmpFileMode is a cmp option that to compare the file mode.
 var CmpFileMode = cmp.Comparer(func(a, b fs.FileMode) bool {
```

Comment:
I think CmpFileMode can be removed. It was only needed because of Windows.

Reply:
fixed by https://github.com/abcxyz/abc/pull/418